test/test_Interface.jl

```diff
@@ -0,0 +1,38 @@
+module TestInterface
+using Compat.Test
+
+# Checking interface through Diversity.jl
+using Diversity
```

Comment:
This is really a testset I guess - I should also move similar tests in SpatialEcology here